schemas/common/descriptors/search/searchdescriptor.schema.json

```diff
@@ -0,0 +1,92 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/searchdescriptor",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Descriptor",
+  "meta:extensible": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor#/definitions/descriptor"
+  ],
+  "meta:abstract": false,
+  "type": "object",
+  "description": "A descriptor that describes search annotations for specific properties of the schemas.",
+  "definitions": {
+    "searchdescriptor": {
+      "properties": {
+        "search:indexable": {
+          "title": "isIndexable",
+          "type": "boolean",
+          "description": " When present, Boolean indicating if the property is indexable / searchable"
```

Comment:
Add the default value (for all entries)

Reply:
Updated with default values for all properties.